test/fpTests/MWC.h

```diff
@@ -0,0 +1,72 @@
+// This tests whether the seed works in single/parallel execution
+
+#include "../../src/baseFunctions/MWC.h"
+
+TEST(setSeed, seed_low)
+{
+    int seed = 123;
+    int curRandNum;
+    int prevRandNum;
+    randomNumberRerFMWC randNum;
+    for (int i = 0; i < 1000; ++i)
+    {
+        randNum.initialize(seed);
+        if (i > 0)
+        {
+            EXPECT_EQ(curRandNum, prevRandNum);
+        }
+        curRandNum = randNum.gen(1000);
+        prevRandNum = curRandNum;
+    }
+}
+
+TEST(setSeed, seed_high)
+{
+    long seed = 2351351351351351;
+    int curRandNum;
+    int prevRandNum;
+    randomNumberRerFMWC randNum;
+    for (int i = 0; i < 1000; ++i)
+    {
+        randNum.initialize(seed);
+        if (i > 0)
+        {
+            EXPECT_EQ(curRandNum, prevRandNum);
+        }
+        curRandNum = randNum.gen(1000);
+        prevRandNum = curRandNum;
+    }
+}
+
+TEST(setSeed, seed_neg)
```

Comment:
Can we check that given a seed the sequence generated is the same over multiple runs?

Reply:
https://github.com/neurodata/packedForest/pull/9/commits/62a3a10d7aaca1c7e5ab6f848cac7d4d3c3bba18